k8s/spark-operator/apptest/deployer/manifest/tester.yaml.template

```diff
@@ -1,25 +1,25 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: "${name}-tester"
+  name: "${APP_INSTANCE_NAME}-tester"
   labels:
-    app.kubernetes.io/name: "${name}"
+    app.kubernetes.io/name: "${APP_INSTANCE_NAME}"
   annotations:
     marketplace.cloud.google.com/verification: test
 spec:
-  serviceAccountName: ${serviceAccount}
+  serviceAccountName: ${SERVICE_ACCOUNT}
   containers:
   - name: tester
     image: "${testerImage}"
     env:
     - name: NAMESPACE
       valueFrom:
         configMapKeyRef:
-          name: ${name}-deployer-config
-          key: namespace
+          name: ${APP_INSTANCE_NAME}-deployer-config
```

Comment:
How about
```

    - name: NAMESPACE
      valueFrom:
        fieldRef:
          fieldPath: metadata.namespace
```

Reply:
I see both in other products. Which one is the preferred one?

Reply:
@jprzychodzen Is the one you suggested the preferred way?

Reply:
Please use metadata for that, it's better than referencing deployer config, no need to add additional reference here.

Reply:
Isn't still done?

Reply:
Done.